From b19b299f4d268f04a2d50809c6cba0a91799ad28 Mon Sep 17 00:00:00 2001
From: Lee Yarwood <lyarwood@redhat.com>
Date: Thu, 27 Jun 2019 11:38:37 +0100
Subject: [PATCH] compute: Correctly pass certified_image_ref as image_id

Ib346d383c430d5151d9aafa6e856dd0a7cae8a23 incorrectly attempted to pass
this as imageRef leading to duplicate arguments being used when calling
the servers client to create the server.

Closes-Bug: #1834461
Change-Id: Ic913176b429b3465dcdbc2e4f44eea32f0b17dfa
(cherry picked from commit cff4ed761da330db1c3611312687f034d812ac98)
---
 tempest/api/compute/servers/test_servers.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tempest/api/compute/servers/test_servers.py b/tempest/api/compute/servers/test_servers.py
index 7429fc4..b1bc075 100644
--- a/tempest/api/compute/servers/test_servers.py
+++ b/tempest/api/compute/servers/test_servers.py
@@ -225,7 +225,7 @@ class ServerShowV263Test(base.BaseV2ComputeTest):
     def test_show_update_rebuild_list_server(self):
         trusted_certs = CONF.compute.certified_image_trusted_certs
         server = self.create_test_server(
-            imageRef=CONF.compute.certified_image_ref,
+            image_id=CONF.compute.certified_image_ref,
             trusted_image_certificates=trusted_certs,
             wait_until='ACTIVE')
 
